internal/app/api/handler/user/user.go

```diff
@@ -63,17 +73,24 @@ func (h *Handler) SignUp(w http.ResponseWriter, r *http.Request) {
 
 // Post handler  login HTTP request
 func (h *Handler) Login(w http.ResponseWriter, r *http.Request) {
+
 	var UserLogin types.UserLogin
+
 	err := json.NewDecoder(r.Body).Decode(&UserLogin)
 
 	if err != nil {
-		respond.JSON(w, http.StatusUnauthorized, h.em.InvalidValue.IncorrectPasswordEmail)
+		respond.JSON(w, http.StatusBadRequest, h.em.InvalidValue.ValidationFailed)
```

Comment:
use if err:= ...; err {} incase no new variable is created
Log error before response
should use another error for case parsing request body failed (Ex; the Request error)